content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -10,27 +10,43 @@ A `rippled` server running in validator mode does everything a stock server does
+It's worth noting that even if your validator isn't a _trusted_ validator, it stills plays an important role in the overall health of the network. These validators help set the standard that trusted validators are measured against. For example, if a trusted validator is disagreeing with a lot of these validators that aren't listed in UNLs, that might indicate a problem.
 
-To run a `rippled` server as a validator, complete the following tasks:
 
-1. Install a `rippled` server. For more information, see [Install rippled](install-rippled.html).
 
-2. [Enable validation on your `rippled` server](#enable-validation-on-your-rippled-server).
+## 1. Understand the traits of a good validator
 
-Beyond this, you can optionally complete the following tasks that help build trust in your validator:
+Strive to have your validator always embody the following properties.
+
+- **Available**
+
+    A good validator is always running and submitting validation votes for every proposed ledger. Strive for 100% uptime.
+
+- **In agreement**
+
+    A good validator's votes match the outcome of the consensus process as often as possible. To do otherwise could indicate that your validator's software is outdated, buggy, or intentionally biased. Always run the [latest `rippled` release](https://github.com/ripple/rippled/tree/master) without modifications. [Watch `rippled` releases](https://github.com/ripple/rippled/releases) to be notified of new releases.
+
+- **Issuing timely votes**
+
+    A good validator's votes arrive quickly and not after a consensus round has already finished. To keep your votes timely, make sure your validator meets the recommended [system requirements](system-requirements.html), which include a fast internet connection.
+
+- **Identified**
+
+    A good validator has a clearly identified owner. Providing [domain verification](#6-provide-domain-verification) is a good start. Ideally, XRP Ledger network UNLs include validators operated by different owners in multiple legal jurisdictions and geographic areas. This reduces the chance that any localized events could interfere with the impartial operations of trusted validators.
+
+Ripple (the company) publishes a [validator list](https://github.com/ripple/rippled/blob/develop/cfg/validators-example.txt) with a set of recommended validators. Ripple strongly recommends using exactly this list for production servers.
 
-1. [Understand the traits of a good validator](#understand-the-traits-of-a-good-validator).
 
-2. [Set up proxies to help protect your validator](#set-up-proxies-to-help-protect-your-validator) from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks.
 
-3. [Provide domain verification](#provide-domain-verification).
+## 2. Install a `rippled` server
 
+For more information, see [Install rippled](install-rippled.html).
```

Comment:
Code font `rippled` here.

The only times you _don't_ want to use code font for `rippled` are:

- Reference links using `[link text][]` syntax (because using code font in this case is buggy)
- Headings containing page titles (because page titles are reused in a variety of contexts that don't always have access to HTML formatting).

This is a good corner case because the title of the page doesn't use code font (for the second reason listed above) but we should still use code font to refer to it.